lib/sentry/event.ex

```diff
@@ -202,7 +204,24 @@ defmodule Sentry.Event do
 
   @spec culprit_from_stacktrace(Exception.stacktrace) :: String.t | nil
   def culprit_from_stacktrace([]), do: nil
-  def culprit_from_stacktrace([{m, f, a, _} | _]), do: Exception.format_mfa(m, f, a)
+  def culprit_from_stacktrace([{m, f, a, _} | _]) do
+    Exception.format_mfa(m, f, arity_to_integer(a))
```

Comment:
Cool, it will convert to `"m.f/n"` rather than `"m.f(a...l...l...a...r...g...s)"`. Thanks again.

Reply:
@redink yep, and the `a...l...l...a...r...g...s` will still show up in the stacktrace on Sentry's website if they are present in the Elixir stacktrace 🙂

Reply:
of course, but it will not appear in `culprit` and will help me for aggregate similar reports.